wiki/Tournaments/UKCC/X/en.md

```diff
@@ -6,13 +6,614 @@ tags:
+| Chemistry ![][flag_GB] | 3 | **5** | ![][flag_GB] **_Snowy** | [#1](https://osu.ppy.sh/community/matches/75664786) |
+| **Malteser** ![][flag_GB] | **7** | 1 | ![][flag_GB] Chugger | [#1](https://osu.ppy.sh/community/matches/75676482) |
+| **Mahmood** ![][flag_GB] | **5** | 3 | ![][flag_GB] Fwuffy | [#1](https://osu.ppy.sh/community/matches/75673884) |
+| **LilyFlower** ![][flag_PH] | **8** | 0 | ![][flag_GB] thereal99gamer | [#1](https://osu.ppy.sh/community/matches/75686078) |
+| **Doomsday** ![][flag_GB] | **6** | 2 | ![][flag_GB] idealnettles | [#1](https://osu.ppy.sh/community/matches/75676510) |
+| **Defex** ![][flag_GB] | **6** | 2 | ![][flag_GB] Kathalshame | [#1](https://osu.ppy.sh/community/matches/75707250) |
+| **Ophiz** ![][flag_IE] | **5** | 3 | ![][flag_GB] Taiga Tiger | [#1](https://osu.ppy.sh/community/matches/75656033) |
+| **Rinnu** ![][flag_GB] | **7** | 1 | ![][flag_GB] [Nathan] | [#1](https://osu.ppy.sh/community/matches/75675970) |
+| **Kerantina** ![][flag_GB] | **7** | 1 | ![][flag_GB] Teejay | [#1](https://osu.ppy.sh/community/matches/75685522) |
+| **SinCrone** ![][flag_GB] | **4** | **4** | ![][flag_GB] **Fozzy** | [#1](https://osu.ppy.sh/community/matches/75696834) |
+| **-Deeline** ![][flag_GB] | **6** | 2 | ![][flag_IE] Xempus | [#1](https://osu.ppy.sh/community/matches/75696811) |
+| HonourBound ![][flag_GB] | 1 | **7** | ![][flag_GB] **Best459** | [#1](https://osu.ppy.sh/community/matches/75765948) |
+
+Sunday, 14 February 2021:
+
+| Player 1 |  |  | Player 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| **Voss** ![][flag_GB] | **6** | 2 | ![][flag_GB] Fozzy | [#1](https://osu.ppy.sh/community/matches/75766280) |
+| **Kingling** ![][flag_GB] | **8** | 0 | ![][flag_IE] Xempus | [#1](https://osu.ppy.sh/community/matches/75757587) |
+| **Jemzsee** ![][flag_GB] | **7** | 1 | ![][flag_GB] thereal99gamer | [#1](https://osu.ppy.sh/community/matches/75768699) |
+| Apo11o ![][flag_GB] | 3 | **5** | ![][flag_GB] **[Nathan]** | [#1](https://osu.ppy.sh/community/matches/75751035) |
+| **Chemistry** ![][flag_GB] | **7** | 1 | ![][flag_GB] idealnettles | [#1](https://osu.ppy.sh/community/matches/75757300) |
+| **Malteser** ![][flag_GB] | **6** | 2 | ![][flag_GB] Matt4132 | [#1](https://osu.ppy.sh/community/matches/75750782) |
+| Mahmood ![][flag_GB] | 3 | **5** | ![][flag_GB] **Zyphox** | [#1](https://osu.ppy.sh/community/matches/75776424) |
+| **LilyFlower** ![][flag_PH] | **5** | 3 | ![][flag_GB] Best459 | [#1](https://osu.ppy.sh/community/matches/75757514) |
+| **Doomsday** ![][flag_GB] | **8** | 0 | ![][flag_GB] squarercube | [#1](https://osu.ppy.sh/community/matches/75744684) |
+| **Defex** ![][flag_GB] | **5** | 3 | ![][flag_GB] Taiga Tiger | [#1](https://osu.ppy.sh/community/matches/75747431) |
+| **Ophiz** ![][flag_IE] | **6** | 2 | ![][flag_GB] Fwuffy | [#1](https://osu.ppy.sh/community/matches/75757629) |
+| Rinnu ![][flag_GB] | -1 | -1 | ![][flag_GB] _Snowy | *forfeit* |
```

Comment:
what does it mean for both players to forfeit? or is this a typo?

Reply:
probably that both players couldn't make it, in which cases they both forfeited their match